src/bp/core/routers/admin/index.ts

```diff
@@ -74,6 +76,14 @@ export class AdminRouter extends CustomRouter {
       res.send(license)
     })
 
+    this.router.post(
```

Comment:
It might be more appropriate to use get here, since we're getting stats from monitoring.

Reply:
It allows to add the from/to time directly in the post request as json, to avoid having a route like monitoring/from/to (and allow additional params in the future, if any). Not a fan of query params